server/sonar-web/src/main/js/apps/quality-gates/components/Condition.tsx

```diff
@@ -105,37 +121,26 @@ export default class Condition extends Component {
     }
 
     if (metric.key.indexOf('new_') === 0) {
-      data.period = '1';
+      data.period = 1;
     }
 
-    e.preventDefault();
-    updateCondition(data)
+    updateCondition({ organization, ...data })
       .then(newCondition => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this function so that its implementation is not identical to the one on line 100. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS4144)